src/transformers/models/auto/configuration_auto.py

```diff
@@ -477,6 +479,7 @@
         ("trajectory_transformer", "Trajectory Transformer"),
         ("transfo-xl", "Transformer-XL"),
         ("trocr", "TrOCR"),
+        ("tvlt", "Tvlt"),
```

Comment:
```suggestion
        ("tvlt", "TVLT"),
```
Please run `make fix-copies` afterwards (might be that it duplicates TVLT in the README's, in which case you can keep one)